File: Wiki/C/code/C_Programming_A_Modern_Approach/ch02_C_Fundamentals/exercises/04.c
// Purpose: study which values are stored in uninitialized variables
// GCC Options: -Wno-uninitialized
// Reference: page 33 (paper) / 58 (ebook)

#include <stdio.h>

    int
main(void)
{
    int a, b, c, d, e;
    float f, g, h, i, j;

    printf("%d,%d,%d,%d,%d\n%f,%f,%f,%f,%f\n",
            a, b, c, d, e, f, g, h, i, j);
    //     0,0,0,0,0
    //     0.000000,0.000000,0.000000,0.000000,0.000000

    return 0;
}

File: Wiki/lua/code/Programming_in_Lua_3rd_edition/ch01_Getting_Started/exercises/04.lua
-- Purpose: Which of the following strings are valid identifiers?
--
--     ___ _end End end until? nil NULL
--
-- Reference: page 8 (paper) / 27 (ebook)

-- They are all valid except:
--
--    - `end` and `nil` because they are reserved keywords
--    - `until?` because it contains an invalid character (`?`); also, `until` is reserved
